docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+services:
+    db:
+        image: mariadb
+        ports:
+        - "3306:3306"
+        environment:
+            MYSQL_ROOT_PASSWORD: prices123
+            MYSQL_DATABASE: aws_prices
+    importer:
+        build: ./importer
+        depends_on:
+        - db
+        links:
+        - db
+        environment:
+            MARIADB_USER: root
+            MARIADB_HOST: db
+        command: ["/scripts/wait-for", "db:3306", "--", "/scripts/pricing_import.py"]
+    server:
+        build: ./server
+        ports:
+        - "4000:4000"
+        depends_on:
+        - db
+        volumes:
+        - ./server:/usr/src/app
```

Comment:
Does this work with relative paths? In the past it hasn't worked unless you specified an absolute path. What's wrong with ${PWD}?

Reply:
Nothing I guess ... I'm actually not sure if it works with relative paths but `${PWD}` would be the same. Like if you ran it from a different directory I'm not sure if `./` would be the directory that you're in or if it's relative to the `docker-compose.yml` file. Either way, `${PWD}` would be wrong too.

Reply:
Right but `./` used to not work with docker mounts...but ${PWD} gives you the absolute path. Sure, both depend on you being in the right place when you execute the command, but the input to the mount is different.